.gitignore

```diff
@@ -59,3 +59,4 @@ target/
 # MacOS
 
 .DS_Store
+cleanco/classify.sublime-workspace
```

Comment:
For gitignore, use a pattern to exclude all sublime workspace files.

Reply:
I just removed it. That's on me.